src/ctia/task/purge_es_stores.clj

```diff
@@ -1,40 +0,0 @@
-(ns ctia.task.purge-es-stores
```

Comment:
this is not maintained nor used anymore.
One can simply launch 
* `DELETE http://localhost:9200/ctia_*` to delete CTIA indices
* `DELETE http://localhost:9200/_template/ctia_*` to delete CTIA indices